admin-manual/maintenance/maintenance.rst

```diff
@@ -377,5 +377,44 @@ known issues with Internet Explorer 11 which result in an inability to start
 transfers in the dashboard (`Issue 7246 <https://projects.artefactual.com/issues/7246>`_). Minimal, but successful,
 testing has been done with Microsoft Edge.
 
+.. _postfix:
+
+Postfix
+_______
+
+The standard mail server setup for Archivematica installs a postfix mail
+service on the same server where the pipeline runs. This Postfix server is
+configured as "Internet Site": A local mail server that sends emails directly
```

Comment:
s/A local/a local

I don't think that you have to start the sentence after the colon with a capital letter.